packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -482,7 +486,7 @@ export class FluidDataStoreRuntime
 		this.notBoundedChannelContextSet.delete(channel.id);
 		// If our data store is attached, then attach the channel.
 		if (this.isAttached) {
-			this.attachChannel(channel);
```

Comment:
Our attach flow is so complicated, I don't know what implications this change does.

Reply:
Good news - it's just a rename :) For more symmetry with other layers